src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Tables/Instance.sql

```diff
@@ -18,6 +18,8 @@ CREATE TABLE dbo.Instance (
     --audit columns
     CreatedDate             DATETIME2(7)               NOT NULL,
     PartitionKey            INT                        NOT NULL DEFAULT 1,  --FK
+    --instance metadata
+    TransferSyntaxUid       VARCHAR(64)                NULL,
```

Comment:
Should we add this AS include column in IX_Instance_StudyInstanceUid_SeriesInstanceUid_Status_PartitionKey ?

Reply:
Good point, IX_Instance_StudyInstanceUid_Status_PartitionKey was also missing SeriesInstanceUID in the include, so I was going let the Keylookup happen. But on a second thought let me fix it.